src/main/java/gregtech/api/metatileentity/SimpleMachineMetaTileEntity.java

```diff
@@ -336,9 +345,9 @@ public void clearMachineInventory(NonNullList<ItemStack> itemBuffer) {
                 .setTooltipText("gregtech.gui.fluid_auto_output.tooltip"));
         }
 
-        builder.widget(new ToggleButtonWidget(rightButtonStartX, 60, 20, 20,
-            GuiTextures.BUTTON_OVERCLOCK, workable::isAllowOverclocking, workable::setAllowOverclocking)
-            .setTooltipText("gregtech.gui.overclock"));
+        builder.widget(new CycleButtonWidget(rightButtonStartX, 60, 20, 20,
```

Comment:
Overclock button should be at least 1 point (maybe more) wider so 3 letters can be rendered without touching border on every GUI scale.